docs/source/recipes/configure-airflow-as-a-runtime.md

```diff
@@ -17,105 +17,127 @@ limitations under the License.
+   ```
+  
+2. Download, review, and customize the [sample `helm` configuration](https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/kubernetes/airflow/helm/values.yaml) (or customize an existing configuration):
+   - Set `git.url` to the URL of the private repository you created earlier, e.g. `ssh://git@github.com/your-git-org/your-dag-repo`
+   - Set `git.ref` to the DAG branch, e.g. `main`.
+   - Set `git.secret` to the name of the secret you created, e.g. `airflow-secret`.
+   - Adjust the `git.gitSync.refreshTime` as desired.
+
+   Example excerpt from a customized configuration:
+
+   ```bash
+   ## configs for the DAG git repository & sync container
+   ##
+   git:
+     ## url of the git repository
+     ##
+     ## EXAMPLE: (HTTP)
+     ##   url: "https://github.com/torvalds/linux.git"
+     ##
+     ## EXAMPLE: (SSH)
+     ##   url: "ssh://git@github.com:torvalds/linux.git"
+     ##
+     url: "ssh://git@github.com/your-git-org/your-dag-repo"
+
+     ## the branch/tag/sha1 which we clone
+     ##
+     ref: "main"
+
+     ## the name of a pre-created secret containing files for ~/.ssh/
+     ##
+     ## NOTE:
+     ## - this is ONLY RELEVANT for SSH git repos
+     ## - the secret commonly includes files: id_rsa, id_rsa.pub, known_hosts
+     ## - known_hosts is NOT NEEDED if `git.sshKeyscan` is true
+     ##
+     secret: "airflow-secret"
+     ...
+     gitSync:
+       ...
+       refreshTime: 10
+   ```
+
+   Note: The example configuration references the pre-built `elyra/airflow` container image, which has the [`airflow-notebook` package](https://pypi.org/project/airflow-notebook/) pre-installed. This package contains the [operator](https://github.com/elyra-ai/airflow-notebook) that supports running of Jupyter notebooks or Python scripts as tasks.
+
+   ```bash
+   airflow:
+   ## configs for the docker image of the web/scheduler/worker
+   ##
+   image:
+     repository: elyra/airflow
+   ```    
+  
+   The container image is created using [this `Dockerfile`](https://github.com/elyra-ai/elyra/tree/master/etc/docker/airflow) and published on [Docker Hub](https://hub.docker.com/r/elyra/airflow) and [quay.io](https://quay.io/repository/elyra/airflow).
+
+3. Install Apache Airflow using the customized configuration.
   
-2. Add the token to your Airflow metadata [runtime configuration](https://elyra.readthedocs.io/en/latest/user_guide/runtime-conf.html)   
+   ```bash
+   helm install "airflow" stable/airflow --values path/to/your_customized_helm_values.yaml
+   ```
+
+Once Apache Airflow is deployed you are ready to create and run pipelines, as described in the [tutorial](../getting_started/tutorials).
+
+## Enabling Elyra pipelines in an existing Apache Airflow deployment
+
+To enable running of notebook pipelines on an existing Apache Airflow deployment  
+- Enable Git as DAG storage
```

Comment:
Should we add a blurb about how to go about this? Something like `On existing installations, this can be configured by modifying your airflow.cfg` in the following area: https://github.com/apache/airflow/blob/6416d898060706787861ff8ecbc4363152a35f45/airflow/config_templates/default_airflow.cfg#L913

Reply:
done